### PR TITLE
Voltage triggers

### PR DESCRIPTION
**Pull request by SwedishSock**
triggers are in and all the scripts for each propp has been made and connected.

### CODE REVIEW

File: Game/Resources/Constant Scene/Minecart/Script/Camera.as [hunk 1]
```diff
@@ -25,9 +25,9 @@ class Camera{
     void Update(float deltaTime) {
         // Only control camera with mouse if we're not running in VR.
         if (!IsVRActive()){
-            //MouseUpdate();
-            self.RotateYaw(0.02f);
-            self.position.y = -2;
+            MouseUpdate();
+            /*self.RotateYaw(0.02f);
+            self.position.y = -2;*/
```
Comment (ludvigarlebrink) [inline Game/Resources/Constant Scene/Minecart/Script/Camera.as]: Out commented code.

File: Game/Resources/Constant Scene/Minecart/Script/Camera.as [hunk 2]
```diff
@@ -25,9 +25,9 @@ class Camera{
...
         }
     }
 }
```
Comment (ludvigarlebrink) [inline Game/Resources/Constant Scene/Minecart/Script/Camera.as]: Missing newline, don't know if it matters in angelscript.

File: Game/Resources/VoltageScene/Scripts/Mastermind.as [hunk 3]
```diff
@@ -55,8 +53,9 @@ class Mastermind {
 	bool puzzleSolved = false;
 	int correct = 0;
 	int rightColors = 0;
+    
 	void CheckSolution(){
-	
+        print("WHOA\n");
```
Comment (ludvigarlebrink) [inline Game/Resources/VoltageScene/Scripts/Mastermind.as]: Print?

File: Game/Resources/VoltageScene/Scripts/Script #22.as [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+    Default(Entity @entity){
+        @hub = Managers();
+        @self = @entity;
+
+        // Remove this if updates are not desired.
+        RegisterUpdate();
+    }
+
+    // Called by the engine for each frame.
```
Comment (ludvigarlebrink) [inline Game/Resources/VoltageScene/Scripts/Script #22.as]: Still think you should remove these comments.

File: Game/Resources/VoltageScene/Scripts/Script #22.as [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    void Update(float deltaTime) {
+
+    }
+}
```
Comment (marredal) [inline Game/Resources/VoltageScene/Scripts/Script #22.as]: When is this script used?

**Pull request merged**